cmake/SuperBuild.cmake

```diff
@@ -18,6 +18,8 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+cmake_policy(SET CMP0114 NEW)

```

Comment:
Only for personal culture, what does it stand for?

Reply:
https://cmake.org/cmake/help/latest/policy/CMP0114.html
mainly a change in the behavior of the dependent steps between several external projects